app/views/templates/MainTemplate.scala.html

```diff
@@ -48,7 +51,7 @@
             timeout = Some(renderConfig.timeoutSeconds),
             countdown = Some(renderConfig.countdownSeconds),
             keepAliveUrl = Some(routes.KeepAliveController.keepAlive(lrn).url),
-            signOutUrl = Some(renderConfig.signOutUrl)
+            signOutUrl = Some(lrn.map(x => routes.DeleteLockController.delete(x).url).getOrElse(renderConfig.signOutUrl))
```

Comment:
Could maybe look to extract this out into its own method in the config with it being used twice?